docs/modules/configuration/pages/dynamic-config.adoc

```diff
@@ -117,7 +117,7 @@ NOTE: If you use the Java member API, you need to use an <<config-file, API>> th
 [[k8]]
 === Persisting Changes in Kubernetes
 
-To persist changes in Kubernetes, you must do the following:
+To persist changes in Kubernetes (K8s), you must do the following:
```

Comment:
Do we need to use the abbreviation `(K8s)`?

Reply:
If users are searching and using K8s then we need to include it in the content ("Speak the user's language") so I'd say Yes, which is why I've sprinkled a few refs. I've never used it **instead** of the full term but just as an addition.